### ISSUE TITLE
Typo in Page - 'Itens'?

### ISSUE THREAD

**Issue by mjamro** [nolleto/Paginator#1]
Shouldn't the name of the property of `Page`  be `Items` instead of `Itens` ?

```
  public class Page<TModel>
    {
        public TModel[] Itens { get; set; }
        public int Index { get; set; }
        public int Length { get; set; }
        public int Count { get; set; }
    }
```

**Comment by nolleto** [comment]
Yes, I confused english by portuguese. My bad =/

**Comment by nolleto** [comment]
Published on NuGet, v2.0

**Comment by mjamro** [comment]
You're quick :) 👍
